source/index.js

```diff
@@ -2,23 +2,25 @@ var Pixi = require("pixi.js")
 var Afloop = require("afloop")
 var Keyb = require("keyb")
 
-import Junkership from "./Junkership.js"
-import Reference from "./Reference.js"
-import Score from "./Score.js"
-import GameContainer from "./GameContainer.js"
-import PowerUp from "./PowerUp.js"
-import Sound from "./Sound.js"
-import Trashbot from "./Trashbot.js"
+import Junkership from "scripts/sprites/Junkership.js"
+import Reference from "scripts/data/Reference.js"
+import Score from "scripts/ui/Score.js"
+import GameContainer from "scripts/sprites/GameContainer.js"
+import PowerUp from "scripts/PowerUp.js"
+import Sound from "scripts/layers/Sound.js"
+import Trashbot from "scripts/sprites/Trashbot.js"
 
 var renderer = Pixi.autoDetectRenderer(Reference.GAME_WIDTH, Reference.GAME_HEIGHT)
 renderer.backgroundColor = 0x222222
 renderer.roundPixels = true
 
 var mount = document.getElementById("mount")
 mount.insertBefore(renderer.view, mount.firstChild)
+
+// Oof.. a global variable??
 window.game = new GameContainer()
```

Comment:
I felt the same way about the `game` global... But it got unwieldy far too quickly. Getting rid of it may be something I look into in the near future.